app/src/pages/ExploreDataPage.tsx

```diff
@@ -104,6 +105,7 @@ function ExploreDataPage() {
 
   return (
     <React.Fragment>
+      <ReactTooltip />
```

Comment:
This is just a nit, but is it possible/easy to have the tooltip anchor to a static point on the button element rather than directly tracking the mouse?

Reply:
I'm not sure! Let's look into it if this is something product decides they want. I agree it's kind of weird that it wiggles about haha